src/Makefile

```diff
@@ -493,7 +493,7 @@ endif
 ### 3.3 Optimization
 ifeq ($(optimize),yes)
 
-	CXXFLAGS += -O3
+	CXXFLAGS += -O3 -fno-tree-slp-vectorize
```

Comment:
that's bad

Reply:
I know & it's outdated, too, please have a look look at the last commit b0e8878 & comment